easybuild/easyconfigs/p/petsc4py/petsc4py-3.20.3-foss-2023b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'petsc4py'
+version = '3.20.3'
```

@tanmoy1989  can you update the version to rely on the PETSc config for foss/2023b that got merged? https://github.com/easybuilders/easybuild-easyconfigs/pull/22696/files#diff-414f478a2655ea5d974a9abfb51d8fc72d99ae6e7de34f405834fd1692d9e084
```suggestion
version = '3.22.5'
```